### PR TITLE
Setup project

### LINKED ISSUE #30: Setup project
Add a simple go server in main.go which serves index.html from port 7777
Add some basic html and css styling to index.html. Use a sans-serif font.
Add a content container with an off-white background. It should have rounded corners and a box shadow.
Inside the content container, advertise a tool called "Pull Pal", which is a digital assistant that monitors your Github repositories and creates pull requests using AI.
Pick a soft color scheme for the page, with light colors. Set the heading font color and background color accordingly.

Files: main.go, index.html

### PR DESCRIPTION
**Pull request by black-crowned-night-heron**
- The main.go file now has a simple server which serves the static files from the "static" directory, which will be created later.
- The index.html file now has some basic HTML and CSS styling.
- A content container with an off-white background, rounded corners, and a box shadow has been added.
- A header with the title "Introducing Pull Pal" has been added, with a blue font color and a light background color.
- A paragraph introducing Pull Pal has been added.

Resolves #30

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        "net/http"
+    )
+    
+    func main() {
+        fs := http.FileServer(http.Dir("static"))
+        http.Handle("/", fs)
+    
+        http.ListenAndServe(":7777", nil)
+    }
+
```
Comment (mobyvb) [inline main.go:13]: remove this unnecessary whitespace at the end

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+    <html>
```
Comment (mobyvb) [inline index.html]: unindent everything below doctype

File: main.go [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package main
+    
+    import (
+        "net/http"
+    )
+    
+    func main() {
+        fs := http.FileServer(http.Dir("static"))
+        http.Handle("/", fs)
```
Comment (mobyvb) [inline main.go]: what does this Handle line do?

File: index.html (hunk 2 above)
Comment (black-crowned-night-heron) [reply index.html]: I have unindented everything below the doctype as you requested.
